spec/facets/SourceCodeJobFacet.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema" : "https://json-schema.org/draft/2020-12/schema",
+  "$id" : "https://openlineage.io/spec/facets/1-0-0/SourceCodeJobFacet.json",
+  "$defs" : {
+    "SourceCodeJobFacet" : {
+      "allOf" : [ {
+        "$ref" : "https://openlineage.io/spec/1-0-2/OpenLineage.json#/$defs/JobFacet"
+      }, {
+        "type" : "object",
+        "properties" : {
+          "language" : {
+            "description" : "Language in which source code of this job was written.",
+            "type" : "string"
+          },
+          "source" : {
```

Comment:
Should we distinguish between `sourceCode` and `sourceFile`?

Reply:
I think the point of this change is that we should have actual code string if we can, instead of pointing to it - which current `SourceCodeLocationJobFacet` should do.

Reply:
I would use `sourceCode ` or `code` to make it clear that is the code string. Just having `source` is a bit confused